ghost/core/core/frontend/helpers/foreach.js

```diff
@@ -31,6 +31,12 @@ module.exports = function foreach(items, options) {
         if (Object.values(items).length > 0 && checks.isPost(Object.values(items)[0])) {
             visibility = visibility || 'all';
         }
+    } if (_.isArray(items) && items.length > 0 && checks.isNewsletter(items[0])) {
```

Comment:
```suggestion
    } 
    
    if (_.isArray(items) && items.length > 0 && checks.isNewsletter(items[0])) {
```

Having the `if` on the same line made it look like it was part of the `else if` which it is not